old/src/Mk-fbx86.mk

```diff
@@ -27,7 +27,7 @@ CFLAGS_LINT = -ansi -pedantic -Wall -Wshadow \
 		-Wmissing-declarations -Wredundant-decls
 
 # Source definitions
-CENVFLAGS = -DCENV_CPU_I386=1 -DCENV_SYS_FREEBSD=1
+CENVFLAGS = -DCENV_CPU_I686=1 -DCENV_SYS_FREEBSD=1
 
```

Comment:
No: This file is unused, and CENV_CPU_I686 doesn't occur anywhere else either.
The other changes to this file are unwanted for the first reason as well.